app/views/works/_meta.html.erb

```diff
@@ -56,11 +56,23 @@
       </dd>
     <% end %>
 
-    <dt class="stats"><%= ts("Stats") %>:</dt>
+    <dt class="stats"><%= ts("Stats:") %></dt>
     <dd class="stats">
 <% end %><!-- end of cache -->
 
       <%= work_meta_list(@work, @chapter) %>
-    </dd>
+      </dd>
+      <% if logged_in_as_admin? %>
+        <dt class="ip">
+          <%= ts('IP Address:') %>
+        </dt>
+        <dd class="ip">
```

Comment:
I will do as you ask, not convinced it feels right though :) as the it then are not seen in the page that is outputted.